### ISSUE TITLE
14.0.1.1 - file app spoiled

### ISSUE THREAD

**Issue by dacoex** [nextcloud/server#11749]
After upgrading to 14.0.1. (stable channel) the file app does not work anymore:

It shows the same page as the log-in pack but with the following message:

```
    File with id "1626" has not been found.

    Storage is temporarily not available

```

This is also a security issue because users cannot logout anymore in this moment!

**Comment by timm2k** [comment]
Update to NC 14.0.2RC2 or wait for NC 14.0.2 release today. This error is fixed in 14.0.2 release.
https://github.com/nextcloud/server/issues/11442

**Comment by MorrisJobke** [comment]
> Update to NC 14.0.2RC2 or wait for NC 14.0.2 release today. This error is fixed in 14.0.2 release.
> #11442

Correct.
